src/test/java/unicash/logic/commands/ResetCommandTest.java

```diff
@@ -33,4 +38,52 @@ public void execute_nonEmptyUniCash_success() {
                 ResetCommand.MESSAGE_SUCCESS, expectedModel);
     }
 
+    @Test
+    public void execute_wrongCommandResult_returnsFalse() {
+        Model model = new ModelManager();
+        Model expectedModel = new ModelManager();
+
+        CommandResult expectedCommandResult =
+                new CommandResult(ResetCommand.MESSAGE_SUCCESS, true, false);
+
+        assertFalse(new ResetCommand().execute(model).equals(expectedCommandResult));
+    }
+
+    @Test
+    public void multipleResetCommand_equalsTrue() {
+        ResetCommand resetCommand = new ResetCommand();
+        assertEquals(resetCommand, new ResetCommand());
+
+    }
+
+    @Test
+    public void sameResetCommand_equalsTrue() {
+        ResetCommand resetCommand = new ResetCommand();
+        assertTrue(resetCommand.equals(resetCommand));
+        assertTrue(resetCommand.equals(new ResetCommand()));
+
+    }
+
+    @Test
+    public void differentCommandTypes_equalsFalse() {
```

Comment:
I don't think this is the right naming convention for unit tests, it should be `method_input_output` I believe

Reply:
Fixed.